applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -144,54 +144,78 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
  */
 static int headset_pres_delay_find(uint8_t index, uint32_t *pres_dly_us)
 {
-	uint32_t pres_dly_min = headsets[index].sink_ep->qos_pref.pd_min;
-	uint32_t pres_dly_max = headsets[index].sink_ep->qos_pref.pd_max;
-	uint32_t pref_dly_min = headsets[index].sink_ep->qos_pref.pref_pd_min;
-	uint32_t pref_dly_max = headsets[index].sink_ep->qos_pref.pref_pd_max;
+	uint32_t pd_min = headsets[index].sink_ep->qos_pref.pd_min;
+	uint32_t pd_max = headsets[index].sink_ep->qos_pref.pd_max;
+	uint32_t pref_pd_min = headsets[index].sink_ep->qos_pref.pref_pd_min;
+	uint32_t pref_pd_max = headsets[index].sink_ep->qos_pref.pref_pd_max;
 
 	LOG_DBG("Index: %d, Pref min: %d, pref max: %d, pres_min: %d, pres_max: %d", index,
-		pref_dly_min, pref_dly_max, pres_dly_min, pres_dly_max);
+		headsets[index].sink_ep->qos_pref.pd_min,
```

Comment:
The print text and values do not match. 
headsets[index].sink_ep->qos_pref.pd_min is printed twice.